generators/entity/index.js

```diff
@@ -633,7 +633,9 @@ class EntityGenerator extends BaseBlueprintGenerator {
                             (this.context.eagerLoad ||
                                 (this.context.paginate !== 'pagination' &&
                                     relationship.relationshipType === 'many-to-many' &&
-                                    relationship.ownerSide === true));
+                                    relationship.ownerSide === true)) &&
```

Comment:
The condition should be OR this.context.databaseType === 'neo4j'?

Reply:
Since the default for Spring Data Neo4J is eager loading, I thought it doesn't make sense to create additional code like findAllWithEager which this flag determines. That's why I turned it to false if the databaseType is neo4j

Reply:
Alternative would be to override the default findAll method with lazy loading and add the eager loading ones. Could do that if you want me to